packages/plugins/src/cli/deploy.ts

```diff
@@ -130,8 +129,6 @@ export const deploy = async (
     process.exit(1)
   }
 
-  const chainId = fetchChainIdForNetwork(network)
```

Comment:
Needed to change the way we fetch the chain ID because we can't assume that the `network` matches the network name in the `SPHINX_NETWORKS`  array